platform-tests/src/test/kotlin/org/eclipse/deeplearning4j/frameworkimport/frameworkimport/onnx/modelzoo/TestPretrainedModels.kt

```diff
@@ -37,9 +38,12 @@ import org.nd4j.shade.protobuf.ProtocolMessageEnum
 import java.io.File
 import java.net.URI
 import java.nio.file.Path
+import javax.annotation.concurrent.NotThreadSafe
 
 data class InputDataset(val dataSetIndex: Int,val inputPaths: List<String>,val outputPaths: List<String>)
 @Tag(TagNames.ONNX)
+@NotThreadSafe
+@Disabled
```

Comment:
When disabling tests, please add a reason why they are disabled and under what circumstances they should be re-enabled.